test/test_automl/test_automl.py

```diff
@@ -47,11 +47,10 @@ def __del__(self):
         pass
 
 
-def test_fit(dask_client, backend):
+def test_fit(dask_client):
```

Comment:
So the overall difference here will be that the temporary directories will be at random while they were given by a pytest fixture before?

Reply:
Also, is anyone taking care that the temporary directories which are now no longer deleted by Auto-sklearn or the fixture are removed?

Reply:
This is a good point. If we do `delete_tmp_folder_after_terminate=False` then they will remain on the temporary directory and will get deleted after the laptop is rebooted. This happens in some test, but not many.

I thought about creating a custom code for deleting this, but as the name is random and did not want to complicate the code.


However I can create a fixture at the begining/end that uses the temporary_directory and clean it if you think we should do this.